RFC-005-SWAP-Basic-Bitcoin.md

```diff
@@ -106,6 +106,62 @@ To compute the exact Bitcoin script bytes of the contract, implementations shoul
 | `refund_identity` | 74                     | 93                    | 20             | See [Parameters](#parameters)                                              |
 | `6888ac`          | 94                     | 96                    | 3              | `OP_ENDIF OP_EQUALVERIFY OP_CHECKSIG`                                      |
 
+
+Script execution overview:
+
+```
+// cur element on stack: byte
```

Comment:
if you want to be precise, you would need to mention all elements on the stack, e.g. current stack
```
<signature>
<pubkey>
<secret>
<01>
```